changes/25307-fleetctl-package-link

```diff
@@ -0,0 +1 @@
+- Add link to information about installing fleetd when generating packages
```

Comment:
Please use past tense :)

Reply:
This is the second time I got the tense wrong, sorry about that!